src/api/public/apidocs-new/paths/source_project_name_project_file.yaml

```diff
@@ -0,0 +1,62 @@
+get:
+  summary: Read a project file
+  description: |
+    Read a project file
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/file_name.yaml'
+    - in: query
+      name: meta
+      required: false
+      schema:
+        type: integer
+      description: Set to 1 to include _meta files
+      example: 1
+    - in: query
+      name: rev
+      required: false
+      schema:
+        type: integer
+      description: Revision number
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/project_meta.yaml'
+          example:
+            name: home:Admin
+            title: Endless Night
+            description: Test project
+            person:
+              userid: Admin
+              role: maintainer
+            repository:
+              name: openSUSE_Tumbleweed
+              path:
+                project: openSUSE.org:openSUSE:Factory
+                repository: snapshot
+              arch:
+                - x86_64
+    '400':
+      description: Wrong value for meta parameter.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            not boolean:
+              value:
+                code: 400
+                summary: "not boolean"
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '403':
+      $ref: '../components/responses/change_project_no_permission.yaml'
```

Comment:
```suggestion
```
This is the GET request. I guess these change project permissions errors are only returned with the PUT endpoint of the next commit.

Reply:
This was discussed offline. The GET request can end up raising [this exception](https://github.com/openSUSE/open-build-service/blob/71355f1d6161139b445a28db1fb799e5b1e88880/src/api/app/models/package/errors.rb#L30). As they aren't exactly the same, it's better not to extract them to the same file.

Each request declares its own response.